### DISCUSSION TITLE
Questions about Copilot + Open Source Software Hierarchy

### DISCUSSION THREAD

**Discussion by liaochris** [hylang/hy#2562] [General]
Hi! My name is Chris and I'm writing a thesis on Open Source Software. I'm trying to collect/validate my data and I have two questions for the maintainers of this project.
1) Did maintainers of this project receive free github copilot access on June 2022? If you don't recall, it might be helpful to check if maintainers are currently elgiible ([https://github.com/github-copilot/free_signup](https://github.com/github-copilot/free_signup))
2) My thesis is especially focused on understanding hierarchical structures. Is there a publicly available list of of individuals in this project with triage/write/maintain/admin access in this project?

Happy to chat further if you have questions, and thank you for your time!

**Comment by Kodiologist** [comment]
Making identical posts on dozens of fora is spam. Don't spam.

**Reply by liaochris** [reply]
I'm just trying to collect data for my thesis, for different projects. If you have a suggestion for a different way to go about it, I'd love to hear it!
